rpc_jobs/irr_role_test.yml

```diff
@@ -0,0 +1,102 @@
+- project:
+    name: 'RPC-IRR-Jobs'
+    # Note: branch is the branch for periodics to build
+    #       branches is the branch pattern to match for PR Jobs.
+    repo:
+      # IRR REPOS ARE ADDED HERE
+      - rpc_maas:
+        irr_repo_url: https://github.com/rcbops/rpc-maas
+    series:
+      - master:
+          branch: master
+          branches: "master"
+    image:
+      - xenial:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+      - trusty:
+          IMAGE: "Ubuntu 14.04 LTS (Trusty Tahr) (PVHVM)"
+    context:
+      - base
+    ztrigger:
+      - pr:
+          CRON: ""
+      - periodic:
+          branches: "do_not_build_on_pr"
+          NUM_TO_KEEP: 10
+    jobs:
+      - 'RPC-IRR_{repo}-{series}-{image}-{context}-{ztrigger}'
+
+- job-template:
+    # DEFAULTS
+    STAGES: >
+      Allocate Resources,
+      Connect Slave,
+      Prepare Deployment,
+      Run IRR Tests,
+      Cleanup,
+      Destroy Slave
+    branch: master
+    NUM_TO_KEEP: 30
+
+    # TEMPLATE
+    name: 'RPC-IRR_{repo}-{series}-{image}-{context}-{ztrigger}'
+    project-type: workflow
+    concurrent: true
+    properties:
+      - build-discarder:
+          num-to-keep: "{NUM_TO_KEEP}"
+      - github:
+          url: {irr_repo_url}
+    parameters:
+      - rpc_gating_params
+      - single_use_slave_params:
+          IMAGE: "{IMAGE}"
+          FLAVOR: "{FLAVOR}"
+          REGION: "{REGION}"
+      - string:
+          name: IRR_CONTEXT
+          default: "{context}"
+      - string:
+          name: IRR_IMAGE
+          default: "{image}"
+      - string:
+          name: IRR_SERIES
+          default: "{series}"
+      - string:
+          name: STAGES
+          default: "{STAGES}"
+          description: |
+            Pipeline stages to run CSV. Note that this list does not influence execution order.
+            Options:
+              Allocate Resources
+              Connect Slave
+              Prepare Deployment
+              Run IRR Tests
+              Cleanup
+              Destroy Slave
+    triggers:
+      - github-pull-request:
+          org-list:
+            - rcbops
+          github-hooks: true
+          trigger-phrase: '.*recheck_cit_all.*|.*recheck_cit_{image}_{context}.*'
```

Comment:
Should this now change to include the whole set of ``{repo}-{series}-{image}-{context}``?